tests/test_build.nf.test.snap

```diff
@@ -191,13 +191,12 @@
                 "cytobands_hg38_GRCh38_v2.4.0.tsv:md5,7bd504feefb33fcfc9be0517439a423c",
                 "protein_domains_hg38_GRCh38_v2.4.0.gff3:md5,43c387a784ebeed71b4147076cebf978",
                 "fusion_report.log:md5,d41d8cd98f00b204e9800998ecf8427e",
-                "fusiongdb2.db:md5,e1ac123a744e515d3e5f85b8344d526a",
-                "mitelman.db:md5,1363795c97f77c641065ecd9ad0e484a",
+                "fusiongdb2.db:md5,715b52d8fa4270c510240c48fb88e9c7",
+                "mitelman.db:md5,51e307dbf4c35c425a3473baa44939c0",
```

Comment:
We should not have the databases md5sums, because the files will change rapidly over time. That includes those two and the hgnc one

Reply:
Thanks for your feedback and letting me know they change rapidly. I will remove them now before merging this one so that the tests are stable.

Reply:
FYI I updated dev with that change

Reply:
We should update the `.nftignore` file, which is the one that allows to ignore md5sum of unstable files :) I can do that now!